src/templates/partials/social/addthis-toolbox.hbs

```diff
@@ -1,2 +1,2 @@
 <!-- Go to www.addthis.com/dashboard to customize your tools -->
-<div class="addthis_sharing_toolbox" data-url="http://www.ceaselessprayer.com" data-title="An Invitation to Joy."></div>
+<div class="addthis_sharing_toolbox" data-url="http://www.ceaselessprayer.com?src=webshare" data-title="Check out this app that helps you pray for others!"></div>
```

Comment:
Can you preview how the text looks in Facebook? Make sure it shows up nice and with a good cover image.